genesis/process/sovereignGenesisBlockCreator.go

```diff
@@ -55,8 +62,27 @@ func (gbc *sovereignGenesisBlockCreator) CreateGenesisBlocks() (map[uint32]data.
 	return gbc.createSovereignHeaders(argsCreateBlock)
 }
 
+func (gbc *sovereignGenesisBlockCreator) initGenesisAccounts() error {
+	acc, err := gbc.arg.Accounts.LoadAccount(core.SystemAccountAddress)
```

Comment:
why are you loading and saving the same accounts without any changes ?

Reply:
This is mandatory at genesis to have accounts initialized.